platform/macos/src/MGLMapView.h

```diff
@@ -247,7 +247,8 @@ MGL_EXPORT IB_DESIGNABLE
  If the value of this property is smaller than that of the `minimumZoomLevel`
  property, the behavior is undefined.
 
- The default value of this property is 20.
+ The default value of this property is 22. The upper bound for this property
+ is 25.5.
```

Comment:
@1ec5 Its nice to know it up front in that case, since there is no other way for a developer to know why its been capped.